src/merge_sort.py

```diff
@@ -0,0 +1,78 @@
+"""Merge Sort Module."""
+
+# MERGE SORT (MS)
+#
+# CodeFellows 401d5
+# Submission Date:
+#
+# Authors:  Colin Lamont <https://github.com/chamberi>
+#           Ben Shields <https://github.com/iamrobinhood12345>
+#
+# URL:
+
+
+def merge_sort(msl):
+    """Merge sort method."""
+    if len(msl) == 1 or not msl:
+        return msl
+    msl1 = msl[:int(len(msl) / 2)]
+    msl2 = msl[int(len(msl) / 2):]
+
+    msl1 = merge_sort(msl1)
+    msl2 = merge_sort(msl2)
+
+    def _merge(msla, mslb):
+        """Merge compares the two lists and returns a sorted list from lowest to highest value.""" 
+        sorted_list = []
+        while len(msla) and len(mslb):
+            if msla[0] < mslb[0]:
+                low = msla.pop(0)
```

Comment:
pop(0) totally works, but I would try and avoid it, as it is an O(n) operation, which kills the efficiency of this sorting algorithm